pyiron_lammps/compatibility/calculate.py

```
@@ -0,0 +1,412 @@
+        else:  # NPT(Nose-Hoover)
+            fix_ensemble_str = "fix ensemble all npt temp {0} {1} {2}".format(
+                str(temperature[0]),
+                str(temperature[1]),
+                str(temperature_damping_timescale),
+            )
+            fix_ensemble_str += pressure_string
+    elif temperature is not None:  # NVT
+        if temperature.min() <= 0:
+            raise ValueError(
+                "Target temperature for fix nvt/npt/nph cannot be 0 or negative"
+            )
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)